README.md

```diff
@@ -33,6 +33,14 @@ If you have [installed conda](https://docs.conda.io/projects/conda/en/latest/use
 conda install -c conda-forge scikit-rf 
 ```
 
+**scikit-rf** includes standalone gui applications which can be installed and used like this
+```Shell
+python -m pip install scikit-rf[plot,qtapps,visa]
```

Comment:
```suggestion
# Install the additional dependencies for the gui applications
python -m pip install scikit-rf[plot,qtapps,visa]
```

Question: I'm discovering this kind of installation method with pip. Is the same kind of install command exist with conda as well?